cppcache/src/ThinClientPoolDM.cpp

```diff
@@ -2414,7 +2415,7 @@ GfErrType FunctionExecution::execute() {
   m_error = m_poolDM->sendRequestToEP(request, reply, m_ep);
   m_error = m_poolDM->handleEPError(m_ep, reply, m_error);
   if (m_error != GF_NOERR) {
-    if (m_error == GF_NOTCON || m_error == GF_IOERR) {
+    if (m_error == GF_NOTCON) {
```

Comment:
how did you determine we no longer need to check for`GF_IOERR`?

Reply:
Removing that check is the way to signal the caller of this function (by means of the return in line 2428) that there has been an IO error so that the NotConnectedException is thrown to the client. With the original check, IO errors were ignored.